Analysis_of_rich_people.R

```diff
@@ -22,5 +76,33 @@ plot <- ggplot(top_10_billionaires, aes(x = reorder(personName, -finalWorth), y
        fill = "Billionaire") +
   theme(axis.text.x = element_text(angle = 45, hjust = 1))
 
-# Show the plot
+# Display the plot
 print(plot)
+
+# The plot displays ten bars in a barchart. Each bar represents the wealth of 
+# one billionaire. The bar only displays the top 10 richest billionaires.
+# The height of the bar represents the wealth of the billionaire in US-dollars.
+
+# ------------------------------------------------------------------------------
+
+# Next, we decided to formulate a thesis. Based on the dataset, we decided that we want
+# to formulate our thesis about the correlation and causation between certain data 
+# columns and the wealth of the billionaires. Maybe this lets us speculate on certain 
+# circumstances that make it easier or harder for someone to become a billionaire.
+
+# In the following we will create multiple plots to compare the correlation between certain
+# data columns and the wealth. Because this dataset only contains billionaires, we won't
+# focus on the difference between their wealth. Being part of this dataset is rich enough ;).
+
+
+
+# ------------------------------------------------------------------------------
+
+# INFO AN DICH FABIAN:
+# SAG MAL WAS DU VON DEM OBEREN ALLES HÄLTST. DIE THESE DA IS ERSTMAL IRGENDWAS. KÖNNEN 
+# WIR GERN ÄNDERN ;)
+
+# ICH HÄTTE ULTRA BOCK EIN PLOT ZU ERSTELLEN; INDEM ICH DEN WOHNORT AUF EINER WELTKARTE ALS 
+# PLOT DARSTELLE. ALSO IRGENDWAS MIT MAP. (falls es dir nichts ausmacht.)
+
+
```

Comment:
Ich schaue mir mal an, wie man das mit quarto macht.
Weil unser Dozent wollte, dass wir das in Quarto prasentieren meine ich.